scripts/config/msu/string.nut

```diff
@@ -7,6 +7,12 @@
 		return first + _string.slice(1);
 	}
 
+	function uncapitalizeFirst( _string )
+	{
+		if (_string == "") return _string;
```

Comment:
One liners rule the world:
`return _string == "" ? _string : _string.slice(0, 1).tolower() + _string.slice(1);`